### PR TITLE
Improve language server error handling

### PR DESCRIPTION
**Pull request by StephenWeatherford**

### CODE REVIEW

**Comment by StephenWeatherford** [comment]
Ping.

BTW, I will fix the build error in a different PR (it's caused by some incomplete code in the base repo).

**Comment by StephenWeatherford** [comment]
Ping

File: src/languageclient/WrappedErrorHandler.ts [hunk 1]
```diff
@@ -24,37 +31,30 @@ export class WrappedErrorHandler implements ErrorHandler {
...
-                stack: parsed.stack
-            },
-            {
-                secondsSinceStart: (Date.now() - serverStartMs) / 1000
-            });
+        // Use our shared error handling code to notification telemetry and user of the error
+        // in a standard way
+        callWithTelemetryAndErrorHandlingSync(languageServerErrorTelemId, (context: IActionContext) => {
+            context.telemetry.properties.jsonrpcMessage = message ? message.jsonrpc : "";
+            context.telemetry.measurements.secondsSinceStart = (Date.now() - this._serverStartTime) / 1000;
```
Comment (PrashanthCorp) [inline src/languageclient/WrappedErrorHandler.ts]: Does it make sense to use stopwatch here? I see it being used in the other PR for the same purpose.

Comment (StephenWeatherford) [reply src/languageclient/WrappedErrorHandler.ts]: Perhaps, but it's a convenience class that really does add much, so I'll leave it for now.  Thanks!

**Pull request merged**